apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -52,41 +52,45 @@ const COW_LISTENERS: CowEventListeners = [
   {
     event: CowEvents.ON_TOAST_MESSAGE,
     handler: (event) => {
-      console.info('[TODO:remove] 🍞 Toast message', event.message, event.data)
-
       // You can provide a simplistic way to handle toast messages (use the "message" to show it in your app)
-      if (event.messageType === ToastMessageType.SWAP_ETH_FLOW_SENT_TX) {
-        console.info('[TODO:remove] 🍞 Toast message: New eth flow order', event.data.tx)
-      }
+      console.info('[TODO:remove][configurator:toast:simple] 🍞 Message: ', event.message)
+    },
+  },
 
-      // ...or you can do handle them your way using the data:
+  {
+    event: CowEvents.ON_TOAST_MESSAGE,
+    handler: (event) => {
+      // You cn implement a more complex way to handle toast messages
```

Comment:
notice I just provide for illustration porpouses 2 ways to implement toast messages.

One is dumb (just show a message), the other is by handling the events